src/main/java/module/Pieces.java

```diff
@@ -0,0 +1,22 @@
+package main.java.module;
+
+public class Pieces {
+    public int getRow() {
+        return row;
+    }
+
+    public void setRow(int row) {
+        this.row = row;
+    }
+
+    public int getCol() {
+        return col;
+    }
+
+    public void setCol(int col) {
+        this.col = col;
+    }
+
+    int row;
```

Comment:
A class should be ordered like this:

Start with the variables
public static constants
followed by private static attributes
followed by private instance attributes
followed by public functions
private utility function just after the public function that calls it.